### PR TITLE
closes #136 Start shift of color scheme

### LINKED ISSUE #136: Change color theme

### PR DESCRIPTION
**Pull request by martensonbj**
Very much still working on this but initial attempt to transition color theme into something more consistant has begun. 

Feedback appreciated! Let me know if you hate colors.

### CODE REVIEW

**Comment by ToniRib** [comment]
Loooove the new front page title!

**Comment by ToniRib** [comment]
Overall I like the new color scheme. For some reason in the cart it looks like the Update and Delete buttons aren't lined up anymore though. I made some minor edits to that and some of the smaller font sizes. Added the file upload option to the admin create item form and did a little styling there also.

**Comment by ToniRib** [comment]
@martensonbj Take a look at the changes and let me know what you think!

File: config/initializers/assets.rb [hunk 1]
```diff
@@ -10,4 +10,4 @@
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
 
-Rails.application.config.assets.precompile += %w( favicon.ico )
+Rails.application.config.assets.precompile += %w( favicon.ico bg1.jpg bg2.jpg bg4 )
```
Comment (martensonbj) [inline config/initializers/assets.rb]: I'm intentionally ignoring this.

**Comment by martensonbj** [comment]
I merged in the cycle background image branch into this branch so now I"m not sure what happens. But the background image should cycle through decently sized images, color scheme has been tweaked, and current master merged in. Let me know if anyone has any further comments!

**Pull request merged**